### ISSUE TITLE
Include sbt extras script

### ISSUE THREAD

**Issue by rnowling** [jove-sh/jove-meta#1]
The [sbt extra script](https://github.com/paulp/sbt-extras) is a useful wrapper script that avoids the extra step of downloading and install sbt.  Including it in jove-meta would make jove easier to consume and build for users.

**Comment by alexarchambault** [comment]
Indeed, but I don't know if it's worth it, as I'm in the midst of deprecating the jove-meta project itself. I'll revive its various parts in a more loosely coupled fashion, easier to test and to maintain, and using [Ammonite](https://github.com/lihaoyi/Ammonite). I already started, see https://github.com/alexarchambault/jupyter-kernel, and https://github.com/alexarchambault/jupyter-scala. I'll give sbt-extra
a try for them.

**Comment by rnowling** [comment]
Thanks for the information!  I'll take a look at them. :)
